src/components/BookItem.jsx

```diff
@@ -13,13 +14,140 @@ const StyledBookItemContainer = styled.li`
+      margin: 0.188rem 20.188rem 0 0;
+      font-family: RobotoSlab;
+      font-size: 1.375rem;
+      font-weight: bold;
+      font-stretch: normal;
+      font-style: normal;
+      line-height: normal;
+      letter-spacing: -0.2px;
+      color: #121212;
+    }
+    .author {
+      width: 6.688rem;
+      height: fit-content;
+      margin: 0.25rem 0 0;
+      font-family: RobotoSlab;
+      font-size: 0.875rem;
+      font-weight: 300;
+      font-stretch: normal;
+      font-style: normal;
+      line-height: normal;
+      letter-spacing: normal;
+      color: #4386bf;
+    }
+    .actions {
+      color: #0290ff;
+      font-size: 0.875rem;
+      font-weight: 300;
+      font-stretch: normal;
+      font-style: normal;
+      line-height: normal;
+      letter-spacing: normal;
+      margin: 20px 0 0;
+      button {
+        background-color: transparent;
+        border: none;
+        outline: none;
+        color: #0290ff;
+        font-size: 0.875rem;
+        font-weight: 300;
+        font-stretch: normal;
+        font-style: normal;
+        line-height: normal;
+        letter-spacing: normal;
+        &:hover {
+          cursor: pointer;
+          text-decoration: underline;
+        }
+      }
+    }
+  }
+  .completed {
+    width: 20%;
+    display: flex;
+    flex-direction: row;
+    span {
+      font-size: 1.5rem;
+      font-weight: bold;
+    }
+  }
+  .vertical-seperator {
+    width: 20%;
+    height: 100%;
+  }
+  .del-button {
+    width: 20%;
+    button {
+      border-radius: 5px;
+      background-color: #0290ff;
+      color: #fff;
+      outline: none;
+      border: none;
+      padding: 10px 20px;
+      &:hover {
+        cursor: pointer;
+        background-color: #0272c4;
+        color: #fff;
+      }
+    }
+  }
+  `;
 
 const BookItem = ({ itemProp, delBook }) => (
   <StyledBookItemContainer>
-    <div>{itemProp.title}</div>
-    <div>{itemProp.author}</div>
-    <button type="button" onClick={() => delBook(itemProp.id)}>Delete</button>
+    <div className="title-container">
+      <div className="category">{itemProp.category}</div>
+      <div className="title">{itemProp.title}</div>
+      <div className="author">{itemProp.author}</div>
+      <div className="actions">
+        <span>Comments</span>
+        <span>  |  </span>
+        <button type="button" onClick={() => delBook(itemProp.id)}>
+          <span>Remove</span>
+        </button>
+        <span>  |  </span>
+        <span>Edit</span>
+      </div>
+    </div>
+    <div className="completed">
+      <div className="complete-loader" />
+      <div style={{ marginLeft: '10px' }}>
+        <span>0%</span>
+        <br />
+        Completed
+      </div>
+    </div>
+    <div className="vertical-seperator">
+      <hr style={{ width: '1px', height: '70%', color: '#e8e8e8' }} />
+    </div>
+    <div className="del-button">
+      <button type="button" onClick={() => delBook(itemProp.id)}>UPDATE PROGRESS</button>
+    </div>
```

Comment:
- [ ] Great job so far @Abdulwhbhussain However, just as the previous reviewer mentioned, your design needs to look like an exact replica of the Zeplin design. However, currently, you only have the `update progress button. You're missing the other two elements on top of the button. So kindly include those in your design and style them as seen in the Figma design :wink: 

| Header | Header |
|--------|--------|
| ![image](https://github.com/Abdulwhbhussain/Bookstore-React-Microverse/assets/102699958/685df28c-4c18-44a3-a089-26453d17577c)|![image](https://github.com/Abdulwhbhussain/Bookstore-React-Microverse/assets/102699958/f19b5088-9925-4808-9819-0bd8f46cef9e)
 |

Reply:
Sure!